### DISCUSSION TITLE
Deployment being failed

### DISCUSSION THREAD

**Discussion by sandipansaha** [breezefront/community#44] [1. Help] [answered]
Hi Roman,

The deployment failed in our server with the below error -

    W:   Higher matching version 2.1.0 of swissup/theme-frontend-breeze-blank was fo  
    W:   und in public repository packagist.org                                       
    W:                                than 2.0.2 in private https://ci.swissuplabs.c  
    W:   om/api/packages.json. Public package might've been taken over by a maliciou  
    W:   s entity,                                                                    
    W:                                please investigate and update package requirem  
    W:   ent to match the version from the private repository  

I can see the new Version 2.1.0 was pushed to the GIT 30 mins ago. Any idea how we can fix the deploy or is it going to be pushed to the private repo soon?

Thanks

**Comment by vovayatsyuk** [comment, marked as answer]
It should be available in a few minutes.

**Reply by sandipansaha** [reply]
Hi @vovayatsyuk 
Seeing this again due to yesterday's new version. Could you please look to push into private repo asap.

  [Exception]                                                                                                                                                       
  Higher matching version 2.3.0 of swissup/module-breeze was found in public repository packagist.org                                                               
                               than 2.2.0 in private https://ci.swissuplabs.com/api/packages.json. Public package might've been taken over by a malicious entity,   
                               please investigate and update package requirement to match the version from the private repository                                   
                                                                                                                                     
Thanks

**Reply by dimdimich** [reply]
This happens because your support and updates subscription is expired most likely and you don't have access to new release from our repository. Easiest way would be to prolong your support and updates subscription or you can simply place new order for free Breeze product at our swissuplabs site.
